samples/request-reply-typescript-greeter/package.json

```diff
@@ -4,13 +4,14 @@
   "description": "Typescript greeting sample",
   "main": "dist/greeter.js",
   "scripts": {
+    "tsc": "tsc",
     "build": "tsc",
     "install": "npm run build",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Scott Andrews",
   "license": "Apache-2.0",
   "dependencies": {
-    "typescript": "^3.4.5"
+    "typescript": "^2.8.1"
```

Comment:
why did all of the sample's with dependencies have the dependency reduced by one major version?